source/cli-translate.js

```diff
@@ -0,0 +1,128 @@
+var imagemagick_cli_lists = null;
```

Comment:
the generated lists are done once. this _will not work_ if the site changes to load content dynamically since it will be too late. This could easily be changed if there is a need.